src/swarm/neo/client/ConnectionSet.d

```diff
@@ -869,12 +870,12 @@ unittest
     ConnectionRegistry!(MockConnection) set;
 
     {
-        assert(set.connection_map.is_empty);
-        assert(set.get(AddrPort.init) is null);
-        assert(set.getBoundary!(false)() is null);
-        assert(set.getBoundary!(true)() is null);
-        assert(set.getNext!(true)(null) is null); // getNext should accept null
-        assert(set.getNext!(false)(null) is null);
+        test(set.connection_map.is_empty);
+        test!("is")(set.get(AddrPort.init), null);
+        test!("is")(set.getBoundary!(false)(), null);
+        test!("is")(set.getBoundary!(true)(), null);
+        test!("is")(set.getNext!(true)(null), null); // getNext should accept null
+        test!("is")(set.getNext!(false)(null), null);
 
         foreach (conn; set)
             assert(false);
```

Comment:
All `assert(false)` should be converted, too, except
```d
switch (++i)
{
    // ...
    case 0: assert(false);
}
```

Reply:
Updated.